### DISCUSSION TITLE
The Discord is pretty hostile, is there a place I can get h elp?

### DISCUSSION THREAD

**Discussion by nafeasonto** [SkriptLang/Skript#5576] [General]
I asked for help on the Discord, and that was a mistake. I am asking for some help here, and a more friendly place if possible? I am new to scripting and trying really hard here, but need some help.

**Comment by NotSoDelayed** [comment]
Make your way to [skUnity Discord](https://discord.gg/skript), as in GitHub is where it houses Skript’s issues, suggestions and pull requests (for contributors).
